utility/create_config.py

```diff
@@ -4,6 +4,6 @@
 
 def create_config(config_path=os.environ['YANGCATALOG_CONFIG_PATH']):
     config = configparser.ConfigParser()
-    config._interpolation = configparser.ExtendedInterpolation() # pyright: ignore
+    config._interpolation = configparser.ExtendedInterpolation()  # pyright: ignore
```

Comment:
Please change this to: 
`config = configparser.ConfigParser(interpolation=configparser.ExtendedInterpolation())`
also change it for all the occurrences in this `backed` repo